tests/integration/humilis-sam-swagger.yaml

```diff
@@ -40,6 +41,7 @@ humilis-sam-testi-swagger:
             paths:
                 - functions:
                     - name: DummyGetResource
+                      description: "Get something"
```

Comment:
To test that method-specific descriptions are indeed added both to the AWS Lambda function, and to the AWS API Gateway method.